### PR TITLE
Added Cloudflare

### PR DESCRIPTION
**Pull request by garrett-wood**
Added Cloudflare with Pricing based on quote.

### CODE REVIEW

**Comment by SaphireLattice** [comment]
Why does it link to something other than CloudFlare?

File: _data/vendors.yml [hunk 1]
```diff
@@ -51,6 +51,14 @@
   pricing_source: https://checklyhq.com/pricing/
   updated_at: 2019-07-30
 
+- name: CloudFlare
+  url: https://cloudsploit.com
```
Comment (Wild1145) [inline _data/vendors.yml:55]: Don't think you updated the link from copying the Cloudsploit one.

**Review by Wild1145 — CHANGES_REQUESTED** [review]

**Comment by abracadabra2** [comment]
Please Re-Add Cloudflare!

**Comment by mbainter** [comment]
This probably should reflect the product. The zero trust product has no SSO tax, so I'm assuming this is one of their other offerings.

**Pull request merged**